attributecert/attributecert.go

```diff
@@ -580,7 +587,14 @@ func parseAttributeCertificate(in *attributeCertificate) (*AttributeCertificate,
 		case attribute.ID.Equal(oidTcgPlatformConfigurationV2):
 			var platformConfiguration PlatformConfigurationV2
 			if _, err := asn1.Unmarshal(attribute.RawValues[0].FullBytes, &platformConfiguration); err != nil {
-				return nil, err
+				var workaround PlatformConfigurationV2_workaround
+				if _, err := asn1.Unmarshal(attribute.RawValues[0].FullBytes, &workaround); err != nil {
+					return nil, err
```

Comment:
Is the error message specific enough that its clear its coming from the workaround unmarshal?

If not, can this be `return nil, fmt.Errorf("parsing workaround: %v", err)`

Reply:
We don't return a specific message for any parsing failures (which is arguably bad but is also how the upstream x509 code behaves). I think fixing that up is probably best done in a different patchset.